docs/public/sdk/GettingStarted.rst

```diff
@@ -220,4 +220,4 @@ The custom log handler can be defined by inheriting from `opentelemetry::sdk::co
         }
     };
     opentelemetry::sdk::common::internal_log::GlobalLogHandler::SetLogHandler(CustomLogHandler());
-    opentelemetry::sdk::common::internal_log::GlobalLogHandler::SetLogLevel(opentelemetry::sdk::common::internal_log::LogLevel::Debug);
\ No newline at end of file
+    opentelemetry::sdk::common::internal_log::GlobalLogHandler::SetLogLevel(opentelemetry::sdk::common::internal_log::LogLevel::Debug);
```

Comment:
what is changed here - spacing?

Reply:
Seems a new line was introduced here which could be removed.